### PR TITLE
Update configure-automated-investigations-remediation.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated navigation to advanced features

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8584

File: microsoft-365/security/defender-endpoint/configure-automated-investigations-remediation.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ To configure automated investigation and remediation:
 
 1. As a global administrator or security administrator, go to the Microsoft 365 Defender portal (<https://security.microsoft.com>) and sign in.
 2. In the navigation pane, choose **Settings**.
-3. In the **General** section, select **Advanced features**.
+3. Click on **Endpoints**, select **Advanced features**.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-automated-investigations-remediation.md]:
```suggestion
3. Select **Endpoints**, then select **Advanced features**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba @JohanFreelancer9

**Pull request merged**